content/design-systems-for-developers/react/en/test.md

```diff
@@ -11,21 +11,21 @@ In chapter 5, we automate design system testing to prevent UI bugs. This chapter
 
 ## Fundamentals of UI component testing
 
-Before we begin, let’s figure out what makes sense to test. Design systems are composed of UI components. Each UI component includes stories (permutations) that describe the intended look and feel given a set of inputs (props). Stories are then rendered by a browser or device for the end-user.
+Before we begin, let’s figure out what makes sense to test. Design systems are composed of UI components, and each UI component includes stories (permutations) that describe the intended look and feel given a set of inputs (props). Stories are then rendered by a browser or device for the end-user.
```

Comment:
```suggestion
Before we begin, let’s figure out what makes sense to test. Design systems are composed of UI components. Each UI component includes stories (permutations) that describe the intended look and feel given a set of inputs (props). Stories are then rendered by a browser or device for the end-user.
```

Revert, the sentence was too long. @jonniebigodes watch out for creating run on sentences in situations like these.